test/view/tournament/tournament_screen_test.dart

```diff
@@ -474,7 +556,7 @@ void main() {
       ]);
       // wait for socket message
       await tester.pump(const Duration(milliseconds: 10));
-      expect(find.byType(PieceWidget), findsNWidgets(32));
```

Comment:
@veloce FYI: After adding the feature game in the tests as well, this expectation failed as it seems to still find the PieceWidgets from the FeatureGame board. So I replaced it with a check to find the name of our opponent instead to verify that the game screen is shown.

Reply:
@tom-anders you could refine by using `find.descendant` to check the pieces are descendant of `GameScreen`.